tests/MongoDB.Driver.Core.Tests/Core/Connections/ConnectionInitializerTests.cs

```diff
@@ -98,9 +98,10 @@ public ConnectionInitializerTests()
         }
 
         [Fact]
-        public void CreateIsMasterCommand_should_return_expected_result()
+        public void AddClientDocumentToIsMasterCommand_with_subject_client_document_should_return_expected_result()
```

Comment:
But you didn't move this test method to the new class?

Reply:
I moved `AddClientDocumentToIsMasterCommand_with_custom_document_should_return_expected_result` to the new class, since that's strictly testing the helper method in IsMasterHelper.

However,  'AddClientDocumentToIsMasterCommand_with_subject_client_document_should_return_expected_result' technically needs a ConnectionInitializer in order to properly run since it's ensuring that a ConnectionInitalizer's _clientDocument is properly added to an isMaster command. It initially looked like this: 
```
        [Fact]
        public void CreateIsMasterCommand_should_return_expected_result()
        {
            var result = _subject.CreateIsMasterCommand();

            var names = result.Names.ToList();
            names.Count.Should().Be(2);
            names[0].Should().Be("isMaster");
            names[1].Should().Be("client");
            result[0].Should().Be(1);
            var clientDocument = result[1].AsBsonDocument;
            var clientDocumentNames = clientDocument.Names.ToList();
            clientDocumentNames.Count.Should().Be(4);
            clientDocumentNames[0].Should().Be("application");
            clientDocumentNames[1].Should().Be("driver");
            clientDocumentNames[2].Should().Be("os");
            clientDocumentNames[3].Should().Be("platform");
            clientDocument["application"]["name"].AsString.Should().Be("test");
            clientDocument["driver"]["name"].AsString.Should().Be("mongo-csharp-driver");
            clientDocument["driver"]["version"].BsonType.Should().Be(BsonType.String);
        }
```

We could remove this test since it's not testing anything that isn't already tested by `AddClientDocumentToIsMasterCommand_with_custom_document_should_return_expected_result`, but I suppose right now it does function as a pseudo-integration test: I could move it IsMasterHelperTests if you'd like.

Reply:
Well according to the name of this method you are testing the AddClientDocumentToCommand method, and that's a method on IsMasterHelper, not ConnectionInitializer.

Reply:
Fair enough, moving the test!